Database/Entities/Faction.cs

```diff
@@ -1,10 +1,32 @@
 ﻿namespace Database.Entities;
 
+
+/// <summary>
+/// Represents an in-game faction to which creatures can belong. <para/>
+/// 
+/// The <c>Faction</c> class provides properties to store and update the unique identifier 
+/// and name of a faction. <para/>
+/// 
+/// <b>Additionally</b>, it includes a collection of associated <c>Creature</c> entities that 
+/// belong to this faction.
+/// </summary>
 public partial class Faction
 {
+    /// <summary>
+    /// Gets or sets the unique identifier.
+    /// This is a 32-bit signed integer.
+    /// </summary>
     public int ID { get; set; }
 
+    /// <summary>
+    /// Gets or sets the faction name.
+    /// </summary>
```

Comment:
As I explained when we talked about this (apart from the "Gets or sets" part) the summary should, ideally, provide more information that can be gathered from the property name.

Better alternative would be:

> How the faction is referred to in the game.